### ISSUE TITLE
Feedback on page SDL2/SDL_RenderSetVSync

### ISSUE THREAD

**Issue by iloveclang** [libsdl-org/sdlwiki#459]
It would be nice if there was some kind of explanation how to check if Vsync has actually been set

because, looking at the code, it appears that the function can only fail if the renderer check fails, or the `int vsync` argument is anything other than 0 or 1.

There appears to be no feedback on whether the vsync actually succeeded.

Is one supposed to check  `if (renderer->wanted_vsync || renderer->simulate_vsync )` ?

**Comment by icculus** [comment]
@slouken, is the vsync results checkable on SDL2? (and SDL3?)

**Comment by slouken** [comment]
No, it's guaranteed to work except on OpenGL, and if it fails in that case we'll simulate vsync, so it always succeeds from the perspective of the application.
